fragments/training/README.md

```diff
@@ -1,3 +1,10 @@
 # Model Training Fragment
 
-**THIS SECTION IS STILL UNDER DEVELOPMENT**
\ No newline at end of file
+The Model Training fragment captures essential information about how the model was trained, including the 
+compute environment, training datasets, and the date on which the model was trained. The goal of this section 
```

Comment:
@duckontheweb are you planning on separate fragments for test/validation (datasets)? Otherwise, it might be a good idea to extend this part with the datasets for test/validation (if they are different from training dataset).

Reply:
Thanks for catching this, I forgot to add a note about the train/test/validation split. One thought I had on this was to utilize the [ML AOI Extension](https://github.com/azavea/stac-ml-aoi-extension/tree/master/ml-aoi) that Azavea has started to develop. That extension is still in the "Proposal" stage, but I've spoken with some of the folks involved and they would be interested in moving it into the `stac-extensions` GitHub org and continuing to develop it there.

I think there are some details to work out in how it would tie in to this spec, so I'll open an issue with a more detailed description.

Reply:
Also, this section in the ADR that Azavea wrote gives a good justification for the ML AOI Extension approach rather than using separate Collections to describe the split (https://github.com/azavea/stac-ml-aoi-extension/blob/master/docs/0003-test-train-validation-split.md#split-by-collection).

Reply:
I realized that using the ML AOI extension actually changes how we structure the training data fields, so I updated the fragment in eb2b26288032b7e5ad1a7bebe7f35a08a2addd15 to reflect this change. Let me know what you think about this approach.

Reply:
@batic I've made some updates to this section based on feedback from @HamedAlemo as well. Let me know if you have any feedback on the changes, and we can also continue to iterate this through issues and new PRs to address how we integrate with the ML AOI Extension.

Reply:
👍